lib/games_teams_collection.rb

```diff
@@ -19,13 +19,39 @@ def total_home_games
     end
   end
 
-  def home_wins
+  def total_home_wins
     @games_teams.count do |game_team|
       game_team.hoa == 'home' && game_team.result == 'WIN'
     end
   end
 
   def percentage_home_wins
-    ((home_wins / total_home_games.to_f) * 100).round(2)
+    ((total_home_wins / total_home_games.to_f) * 100).round(2)
+  end
+
+  def total_away_games
+    @games_teams.count do |game_team|
+      game_team.hoa == 'away'
+    end
+  end
+
+  def total_away_wins
+    @games_teams.count do |game_team|
+      game_team.hoa == 'away' && game_team.result == 'WIN'
+    end
+  end
+
+  def percentage_away_wins
+    ((total_away_wins / total_away_games.to_f) * 100).round(2)
+  end
+
+  def total_ties
+    @games_teams.count do |game_team|
+      game_team.result == 'TIE'
+    end
+  end
+
+  def percentage_ties
+    ((total_ties.to_f / @games_teams.count) * 100).round(2)
```

Comment:
I believe `@games_teams.count` will be grabbing two of every game since there's a row for each team in each game (because it's a join table like we talked about earlier). What could work in its place is a method that grabs all the `game_id` values and uses `.uniq` to remove the duplicates, then `.count` that collection; that's the second way of getting a total number of games I mentioned this afternoon. Some of the methods I wrote in `games_collection.rb` (namely, `every`, `every_unique`, and `total_unique`) are for that exact purpose and could be copied and pasted here if you wanted to save some time. Otherwise you could just sub in `total_away_games`, which would give you the same result at least in the dummy file.

Reply:
If we take the ties divided by the total games, we get the appropriate ration because there are 2 ties per game, whereas there is only 1 win per game.